inference_cli/main.py

```diff
@@ -14,6 +14,26 @@
 app.add_typer(benchmark_app, name="benchmark")
 
 
+def version_callback(value: bool):
+    if value:
+        from importlib.metadata import version, PackageNotFoundError
+
+        try:
+            inference_version = version('inference')
+            print(f"Inference Version: {inference_version}")
+        except PackageNotFoundError:
+            print(f"Inference Version: using local install")
+            
+        raise typer.Exit()
+
+@app.callback()
+def main(
```

Comment:
Please rename this method so the name states the purpose behind this method (maybe `version`?)